website/content/en/v0.30/concepts/deprovisioning.md

```diff
@@ -216,7 +216,7 @@ Voluntary node removal does not include [Interruption]({{<ref "#interruption" >}
 
 ### Node-Level Controls
 
-Nodes can be opted out of consolidation deprovisioning by setting the annotation `karpenter.sh/do-not-consolidate: "true"` on the node.
+Nodes can be opted out of consolidation and expiration deprovisioning by setting the annotation `karpenter.sh/do-not-consolidate: "true"` on the node.
```

Comment:
I don't think that we want to make this change in docs versions prior to v0.32 since this annotation wasn't available in Karpenter yet

Reply:
Removed from v0.30 and v0.31.